src/main/java/io/vertx/core/http/HttpServerResponse.java

```diff
@@ -371,4 +371,8 @@ default HttpServerResponse sendFile(String filename, long offset, Handler<AsyncR
   @Fluent
   HttpServerResponse bodyEndHandler(@Nullable Handler<Void> handler);
 
+  /**
+   * @return the total number of bytes written for this response
```

Comment:
Are you talking about the comment, which I agree is misleading, or the name of the method too? The method name was originally suggested by @purplefox, so may want his 2 cents.